index.js

```diff
@@ -2295,25 +2320,68 @@
     impl: isRight
   };
 
-  //# fromEither :: b -> Either a b -> b
+  //# fromLeft :: a -> Either a b -> a
+  //.
+  //. Takes a default value and an Either, and returns the Left value
+  //. if the Either is a Left; the default value otherwise.
+  //.
+  //. See also [`fromRight`](#fromRight).
+  //.
+  //. ```javascript
+  //. > S.fromLeft (0) (S.Left (42))
+  //. 42
+  //.
+  //. > S.fromLeft (0) (S.Right (42))
+  //. 0
+  //. ```
+  function fromLeft(x) {
+    return either (I) (K (x));
+  }
+  _.fromLeft = {
+    consts: {},
+    types: [a, $.Either (a) (b), a],
+    impl: fromLeft
```

Comment:
What do you think of using `B (either (I)) (K)` here?

Reply:
I don't see a difference, except perhaps stylistic, so you decide.

Reply:
Let's stick with your implementation. :)